nodejs/week2/homework/warmup/index.js

```diff
@@ -0,0 +1,46 @@
+const express = require('express');
+const app = express();
+
+function numbers(req, res) {
+    const firstNumber = Number(req.query.a);
+    const secondNumber = Number(req.query.b);
+    return firstNumber, secondNumber;
+}
+
+function add(req, res) {
+    numbers(req, res);
+    const answer = firstNumber + secondNumber;
+    if (firstNumber && secondNumber) {
+        res.send('add result: ' + answer);
+    }
+    if (firstNumber) {
+        res.send('you typed first number only: ' + firstNumber);
+    }
+    if (secondNumber) {
+        res.send('you typed second number only: ' + secondNumber);
+    }
+}
+console.log(add(5, 5));
```

Comment:
Can you explain what you're doing in this function?
The parameter names are a little confusing